src/CMakeLists.txt

```diff
@@ -27,6 +27,11 @@ if (WIN32)
 
 elseif (UNIX)
   add_executable(${EXECUTABLE_TITLE} ${SOURCE_FILES})
+  if(WITH_UCHECKMARKS)
+    # Do nothing
+  else()
+    target_compile_options(${EXECUTABLE_TITLE} PUBLIC "-DNOUCHECKMARKS")
```

Comment:
Remove the `-D` prefix, it's not necessary.

Reply:
It is, removeing it results in ... ```-pthread  -DNDEBUG NOUCHECKMARKS```

Reply:
You're right. I had it confused with `target_compile_definitions`. I think it would be better to use that. See [CMake documentation](https://cmake.org/cmake/help/latest/command/target_compile_definitions.html). Please let me know if that works instead.

Reply:
@diizzyy Please change this to:
```CMake
target_compile_definitions(${EXECUTABLE_TITLE} PUBLIC "NOUCHECKMARKS")
```
then I will merge it.

Reply:
Updated